src/components/MagicForm/MagicForm.js

```diff
@@ -1,107 +1,74 @@
 import styled from 'styled-components';
 
-export default function MagicForm({analyzeSpell}) {
-  const [savedOrders, SetSavedOrders] = useState([]);
+import {AriaOnlySpan, FrameSpan} from '../Stylesheet/StyledSpans.js';
 
+export default function MagicForm({saveSpellOrder, savedOrders}) {
   function handleSpell(event) {
     event.preventDefault();
     const Form = event.target;
     const {input} = Form.elements;
-    //is already trimmed() here
-    const checkedInput = input.value.toLowerCase();
-    if (checkedInput.length > 0) {
-      const inputInfo = analyzeSpell(checkedInput);
-      SetSavedOrders([{value: input.value, info: inputInfo}, ...savedOrders]); //input.value means the original string 1:1
+
+    if (input.value.length > 0) {
+      saveSpellOrder(input.value);
       Form.reset();
-    } //Get an empty string before it runs into analyzeSpell
-    else {
-      SetSavedOrders([
-        {
-          value: '...oh nothing?',
-          info: 'Did you fell on your enter-button? you have to type in the commands to get information about zshell!',
-        },
-        ...savedOrders,
-      ]);
     }
+
+    input.focus();
   }
 
   return (
-    <OrganizingDiv>
-      <SavedOrdersList role="list">
-        {savedOrders.map(order => (
-          <SavedOrderListitem key={nanoid()}>
-            <AriaOnlySpan>you</AriaOnlySpan>
-            typed in: {order.value}
-            <p>{order.info}</p>
-          </SavedOrderListitem>
-        ))}
-      </SavedOrdersList>
-
-      <Form onSubmit={handleSpell} role="form">
-        <label htmlFor="input">
-          <AriaOnlySpan>type in your command</AriaOnlySpan>
-        </label>
-        <Button> ❯ ❚ </Button>
-        <Input id="input" name="input" autoComplete="off" />
-      </Form>
-    </OrganizingDiv>
+    <Form onSubmit={handleSpell} role="form">
+      <label htmlFor="input">
+        <AriaOnlySpan>type in your command</AriaOnlySpan>
+      </label>
+      <CommandLineSpanL>❯ ❚ </CommandLineSpanL>
+      <Input id="input" name="input" autoComplete="off" />
+      <Button>ENTER</Button>
+      <CommandLineSpanR>
+        <FrameSpan>?</FrameSpan> {savedOrders.length}
+      </CommandLineSpanR>
+    </Form>
   );
 }
 
-const OrganizingDiv = styled.div`
-  width: 65vw;
-  max-width: 500px;
-  height: auto;
-  max-height: 500px;
-  border: 1px solid black;
-  border-radius: 5%;
-`;
-
-const SavedOrdersList = styled.ul`
-  border: 1px solid black;
-  list-style-type: 'you ';
-  height: 50vw;
-  max-height: 400px;
-  overflow: scroll;
+const Form = styled.form`
   display: flex;
-  flex-direction: column-reverse;
   justify-content: flex-start;
-  padding-bottom: 0;
-  margin-bottom: 0;
-`;
-
-const SavedOrderListitem = styled.li`
-  word-wrap: break-word;
-  padding-top: 1rem;
-  border-top: 2px solid black;
+  align-items: baseline;
+  flex-wrap: nowrap;
+  margin-bottom: 1rem;
 `;
 
 const Input = styled.input`
   width: 90%;
+  border: none;
+  outline: none;
```

Comment:
Please don't set the `outline` to `none`. This removes the focus on the input elements and can cause accessibility issues.